### DISCUSSION TITLE
paste as plain text

### DISCUSSION THREAD

**Discussion by vc-chirag** [sibiraj-s/ngx-editor#541] [General]
When I paste a text such as **user@mail.com**, the output appears as  `<p>user<a href=\"mail.com\" target=\"_blank\">mail.com</a></p>"`. However, when I manually typing the **user@mail.com** so the output is in plain text. why there a difference between? 
Any help would be greatly appreciated! @sibiraj-s

**Comment by sibiraj-s** [comment]
It totally depends on the context where you are pasting it. If you are pasting on a new line or in a empty editor, the `p` tag is added, in between of the sentence `p` tag won't be added, also the pasted content depends on where you copy the content from(the source).

Try right click to copy just the email and paste it to see if you are still getting the same behavior.

**Reply by vc-chirag** [reply]
Thank you for your quick response, , The issue lies in the automatic generation of an `<a>` for redirection when **test@mail.com** is added, which I want to avoid. And  I m pasting this content is from notepad. 
Is there any flag or configuration is available to enable paste-as-plain-text option in editor?
